validator/build.py

```diff
@@ -284,7 +284,7 @@ def CompileWithClosure(js_files, definitions, entry_points, output_file):
 
   cmd = [
       'java', '-jar', 'node_modules/google-closure-compiler-java/compiler.jar',
-      '--language_out=ES5_STRICT', '--dependency_mode=STRICT',
+      '--language_out=ES5_STRICT', '--dependency_mode=PRUNE',
```

Comment:
What's the difference?

Reply:
`STRICT` appears to have been deprecated. See [this](https://travis-ci.org/ampproject/amphtml/jobs/636642185#L366-L380) error.

In any case, we use `PRUNE` while running `gulp dist`, so I think we should use it while running the validator tests as well.

Reply:
Yes, the [release notes](https://github.com/google/closure-compiler/wiki/Releases#january-1-2020-v20200101) confirm that `STRICT` has been removed in favor of `PRUNE`. We're good here.

Reply:
https://github.com/google/closure-compiler/commit/0c8ae0ec822e89aa82f8b7604fd5a68bc30f77ea#diff-17c34c561974efe2470d59d8ba5f4efaL164, looks like they are equivalent.